website/docs/cloud-docs/api-docs/project-team-access.mdx

```diff
@@ -367,3 +486,23 @@ $ curl \
   --request DELETE \
   https://app.terraform.io/api/v2/team-projects/tprj-WbG7p5KnT7S7HZqw
 ```
+
+## Implied Custom Permission Levels
+
+As mentioned above, when setting team access levels of `read`, `write`, `maintain`, or `admin`, the following permissions that can be set individually when using an access level of `custom`, are given values based on the access level.
```

Comment:
Oops, I skipped this file in my first review!

I suggest shortening to (I think this still covers the same information):
```suggestion
As mentioned above, when setting team access levels (`read`, `write`, `maintain`, or `admin`), you can individually set the following permissions if you use the `custom` access level.
```